pkg/resources/gce/gce.go

```diff
@@ -377,10 +377,10 @@ func deleteGCEDisk(cloud fi.Cloud, r *resources.Resource) error {
 	op, err := c.Compute().Disks().Delete(u.Project, u.Zone, u.Name)
 	if err != nil {
 		if gce.IsNotFound(err) {
-			klog.Infof("disk not found, assuming deleted: %q", t.SelfLink)
+			klog.Infof("Disk not found, assuming deleted: %q", t.SelfLink)
 			return nil
 		}
-		return fmt.Errorf("error deleting disk %s: %v", t.SelfLink, err)
+		return fmt.Errorf("error deleting Disk %s: %v", t.SelfLink, err)
```

Comment:
Error messages should not start with the word "error"

```suggestion
		return fmt.Errorf("deleting Disk %s: %v", t.SelfLink, err)
```

Reply:
sorry... missed this

Reply:
I'll do it in a followup PR